chainstate/storage/src/lib.rs

```diff
@@ -71,8 +73,11 @@ pub trait BlockchainStorageRead: UtxosStorageRead {
     /// Get token creation tx
     fn get_token_aux_data(&self, token_id: &TokenId) -> crate::Result<Option<TokenAuxiliaryData>>;
 
-    // Get token id by id of the creation tx
+    /// Get token id by id of the creation tx
     fn get_token_id(&self, tx_id: &Id<Transaction>) -> crate::Result<Option<TokenId>>;
+
+    /// Get token id by id of the creation tx
+    fn get_block_tree_by_height(&self) -> crate::Result<BTreeMap<BlockHeight, Vec<Id<Block>>>>;
```

Comment:
The comment is from a different function. 🙃

Reply:
Thanks. Fixed.